packages/payments-client/src/components/payment/payment.tsx

```diff
@@ -174,13 +173,13 @@ export const Payment: FC<PaymentProps> = ({ paymentRequest }) => {
     method: 'POST',
     returnType: UpdateReturnTypeEnum.RESPONSE,
     headers: {
-      Authorization: idToken,
-      'reapit-customer': clientCode as string,
+      'reapit-customer': clientCode,
+      'reapit-id-token': idToken,
       'reapit-app-id': process.env.appId,
     },
   })
 
-  useEffect(handleGetMerchantKey(getMerchantKey, idToken, config), [config, idToken])
+  useEffect(handleGetMerchantKey(getMerchantKey, idToken, config), [idToken, config])
```

Comment:
Just checking, did you mean to swap these values here?

Reply:
I removed the idToken then had to add it back, long day!

Reply:
It's just a useEffect dependency array - the order doesn't matter